lib/modules/arguments-widgets/views/reaction.html

```diff
@@ -1,6 +1,10 @@
 <div class="reaction argument-variant-{{arg.label}}" id="arg{{reaction.id}}">
   <div class="user {{'admin' if  data.isAdmin}}">
-    <strong>{{reaction.user.firstName}} {{reaction.user.lastName}}</strong> | {{reaction.createdAt | date('LLL')}}
+    <strong>{{reaction.user.firstName}}
+      {% if data.widget.showLastNameForReactions == 'yes' or (data.widget.showLastNameForReactions == 'adminonly' and reaction.user.role == 'admin') %}
```

Comment:
Misschien deze logica in node afhandelen zodat het template leesbaar en simpel blijft. Dus dan krijg je zoiets:
```if (data.widget.showLastName())```

Reply:
Heb ik toegevoegd! Helaas moet je aan een helper functie wel de data.widget en de user meegeven. Voelt daardoor nog niet helemaal ideaal.